### ISSUE TITLE
Training questions about the train part of main res.py  

### ISSUE THREAD

**Issue by straw66** [ThomasWestfechtel/SKD#1]
Hi author, I would like to ask why GPU utilization is high at the beginning of Training, but when print (Start Training!) After that, GPU utilization dropped dramatically and I changed all the num_workers in main_res to 16 and batch_size to 64 to see what might be wrong and how to fix it. The device I use is RTX3070.
Screenshot of the initial runtime：
![1fc412b38d471e4700094b9ab4656539](https://github.com/ThomasWestfechtel/SKD/assets/99629643/16d025a6-0704-429a-abf0-fbeb57d702ca)
When print (Start Training!) Screenshot of the situation afterwards：
![e1314310922830676cb7b3cc011a6164](https://github.com/ThomasWestfechtel/SKD/assets/99629643/856176fd-3843-400e-be62-1665474b22bc)

**Comment by ThomasWestfechtel** [comment]
My guess is that it is due to the reparametrization of the batchnorm layers of the ResNet backbone.
To test this you could change the backbone to ViT-base, ViT doesn't utilize batchnorm layers, and therefore the spike in GPU utilization shouldn't happen for it.
